tests/pytype_test.py

```diff
@@ -70,7 +70,7 @@ def pytype_test(args):
         print("Cannot run pytd. Did you install pytype?")
         return 0, 0
 
-    wanted = re.compile(r"stdlib/(2|2\.7|2and3)/.*\.pyi$")
+    wanted = re.compile(r"stdlib/(2|2\.7|2and3)/.*\.pyi?$")
```

Comment:
The existing `pytype_test.py` checks `pyi` files using the `pytd` tool. If you want to run against `py` files, you'll need to run the `pytype` executable.

Sorry. I should have caught that earlier.